defaults/main.yml

```diff
@@ -21,46 +21,50 @@ graylog_plugin_dir:           '/usr/share/graylog-server/plugin'
 graylog_root_email:           ''
 graylog_root_timezone:        'UTC'
 graylog_alert_check_interval: 60
-graylog_disable_sigar:        'False'
+graylog_disable_sigar:        'false'
 
 # REST
 graylog_rest_listen_uri:              'http://0.0.0.0:9000/api/'
 graylog_rest_transport_uri:           ''
-graylog_rest_enable_cors:             'True'
-graylog_rest_enable_gzip:             'True'
-graylog_rest_enable_tls:              'False'
-graylog_rest_tls_cert_file:           '/path/to/graylog.crt'
-graylog_rest_tls_key_file:            '/path/to/graylog.key'
-graylog_rest_tls_key_password:        'secret'
+graylog_rest_enable_cors:             'true'
+graylog_rest_enable_gzip:             'true'
+graylog_rest_enable_tls:              'false'
+graylog_rest_tls_cert_file:           ''
+graylog_rest_tls_key_file:            ''
+graylog_rest_tls_key_password:        ''
 graylog_rest_max_header_size:         8192
 graylog_rest_max_initial_line_length: 4096
 graylog_rest_thread_pool_size:        16
 
 # Search
-graylog_elasticsearch_template_name:                    'graylog-internal'
-graylog_allow_leading_wildcard_searches:                'False'
-graylog_allow_highlighting:                             'False'
-graylog_elasticsearch_config_file:                      ''
-graylog_elasticsearch_shards:                           4
-graylog_elasticsearch_replicas:                         0
-graylog_elasticsearch_index_prefix:                     'graylog'
-graylog_elasticsearch_cluster_name:                     'graylog'
-graylog_elasticsearch_node_name_prefix:                 'graylog-'
-graylog_elasticsearch_transport_tcp_port:               9350
-graylog_elasticsearch_discovery_zen_ping_unicast_hosts: '127.0.0.1:9300'
-graylog_elasticsearch_network_host:                     ''
-graylog_elasticsearch_network_bind_host:                ''
-graylog_elasticsearch_network_publish_host:             ''
-graylog_elasticsearch_analyzer:                         'standard'
-graylog_elasticsearch_output_batch_size:                25
-graylog_elasticsearch_output_flush_interval:            1
-graylog_elasticsearch_disable_version_check:            'True'
-graylog_elasticsearch_http_enabled:                     'False'
-graylog_disable_index_optimization:                     'True'
-graylog_index_optimization_max_num_segments:            1
+graylog_elasticsearch_hosts:                           'http://127.0.0.1:9000/api/'
+graylog_elasticsearch_connect_timeout:                 ''
```

Comment:
These timeout vars need to be empty otherwise they're causing
```
Exception in thread "main" java.lang.ClassCastException: com.github.joschi.jadconfig.util.Duration cannot be cast to java.lang.Integer
```